app/models/calculation/CalculationResponseModel.scala

```diff
@@ -37,7 +39,12 @@ object CalculationResponseModel {
   implicit val format: OFormat[CalculationResponseModel] = Json.format[CalculationResponseModel]
 }
 
-case class Metadata(calculationTimestamp: Option[String], crystallised: Option[Boolean], calculationReason: Option[String])
+case class Metadata(
+                     calculationTimestamp: Option[String],
```

Comment:
just wondering why not to use calculationTimestamp: Option[LocalDate]?

Reply:
the calculation period dates  are going to be coming from Backend and the values will the Standard Quarters. Thats what is decided for MISUV-3880

Reply:
I can make the change. I don't see any problem.